modules/log6x-quickstart-viaq.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-about.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="quick-start-viaq_{context}"]
+= Quick start with ViaQ
+
+To use the default ViaQ data model, follow these steps:  
+
+.Prerequisites
+* Cluster administrator permissions
+
+.Procedure
+
+. Install the {clo}, {loki-op}, and {coo-first} from OperatorHub.
+
+. Create a `LokiStack` custom resource (CR) in the `openshift-logging` namespace:
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  managementState: Managed
+  size: 1x.extra-small
+  storage:
+    schemas:
+    - effectiveDate: '2024-10-01'
+      version: v13
+    secret:
+      name: logging-loki-s3
+      type: s3
+  storageClassName: gp3-csi
+  tenants:
+    mode: openshift-logging
+----
++
+[NOTE]
+====
+Ensure that the `logging-loki-s3` secret is created beforehand. The contents of this secret vary depending on the object storage in use. For more information, see Secrets and TLS Configuration.
+====
+
+. Create a service account for the collector:
++
+[source,terminal]
+----
+$ oc create sa collector -n openshift-logging
+----
+
+. Allow the collector's service account to write data to the `LokiStack` CR:
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user logging-collector-logs-writer -z collector
+----
++
+[NOTE]
+====
+The `ClusterRole` resource is created automatically during the Cluster Logging Operator installation and does not need to be created manually.
+====
+
+. Allow the collector's service account to collect logs:
++
+[source,terminal]
+----
+$ oc project openshift-logging
+----
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-application-logs -z collector
+----
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-audit-logs -z collector
+----
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z collector
+----
++
+[NOTE]
+====
+The example binds the collector to all three roles (application, infrastructure, and audit), but by default, only application and infrastructure logs are collected. To collect audit logs, update your `ClusterLogForwarder` configuration to include them. Assign roles based on the specific log types required for your environment.
```

Comment:
"but by default, only application and infrastructure logs are collected". there isn't default inputs values in 6.0.  I think we can remove the word 'by default'

Reply:
The example binds the collector to all three roles (application, infrastructure, and audit) and collect application and infrastructure logs.

Reply:
I think we needn't explain why we didn't collect audit logs in this example. that may confuse the customer.

Reply:
Updated the Note as, "The example binds the collector to all three roles (application, infrastructure, and audit) to collect application and infrastructure logs. Assign roles based on the specific log types required for your environment."